plugin/src/main/java/org/wso2/carbon/uuf/maven/bean/AppConfig.java

```diff
@@ -122,6 +123,24 @@ public void setLoginPageUri(String loginPageUri) {
     }
 
     /**
+     * Get session manager implementation class.
+     *
+     * @return session manager implementation class
+     */
+    public String getSessionManager() {
+        return sessionManager;
+    }
+
+    /**
+     * Set session manager implementation class.
+     *
+     * @param sessionManager session manager implementation class
+     */
+    public void setSessionManager(String sessionManager) {
```

Comment:
We need a test for the `sessionManager` configuration. Value of `sessionManager` can be `null` or a non-empty string. For invalid values (e.g. empty string) we should throw `IllegalArgumentException`. Please refer [AppConfigTest.testThemeValidations()](https://github.com/wso2/carbon-uuf-maven-tools/blob/v1.0.0-m13/plugin/src/test/java/org/wso2/carbon/uuf/maven/bean/AppConfigTest.java#L43) test case.